content/assignments/techjobs-console/_index.md

```diff
@@ -0,0 +1,477 @@
+for programs to read in.
+
+If CSV files are new to you, don't worry, they're easy to understand.
+CSV files are conceptually similar to simple spreadsheets in that they
+organize data in rows and columns. The major difference is that they
+don't have the ability to carry out calculations the way spreadsheets
+do, and you can easily open, read, and edit them in plain text editors.
+
+Open up `jobs_data.csv`, which is in the project. You'll see that the first line is:
+
+```console
+   name,employer,location,position type,core competency
+```
+
+While it isn't required, the first line of a CSV file often represents
+the column names. We have 5 names here, which indicates that each of our
+rows in the CSV file should have 5 fields. In this file format, a **row**
+corresponds to a new line. So each line below the first will constitute
+a row of data, or a record.
+
+Have a look at the data below line 1, and ask yourself the following
+questions:
+
+1. Which fields match up with which column names above?
+1. Why do some lines/rows (e.g. line 10) have more commas than others, if
+   commas are supposed to separate columns?
+1. What role do the double-quotes play in lines 10 and 79?
+
+## The TechJobs Class
+
+The `TechJobs` class contains the method that will drive our
+program's functionality. It contains three methods:
+
+1. `RunProgram()` - The main application runner.
+1. `GetUserSelection()` - A utility method that displays a menu of choices and
+   returns the user's selection.
+1. `PrintJobs()` - This is meant to print a list of jobs to the console in a
+   nicely formatted manner, but hasn't been implemented yet. This will be part
+   of your job.
+
+Let's look at each of these.
+
+### The `RunProgram()` Method
+
+The logic within `RunProgram()` presents menus in turn, and based on the
+user's choice, takes appropriate action.
+
+It begins by declaring two local variables: `actionChoices` and
+`columnChoices`. These contain information relating to the menus that
+we'll display, and we'll look at them in more detail later.
+
+Next, we notice a `while loop` that starts `while (true)`. While we usually
+want to avoid creating infinite loops, we have a good reason for doing so in
+this case! We want our application to continually run until the user decides
+they want to quit. The simplest way to do this is to loop forever. When the
+user wants to quit our app, they can enter `x` at the initial `View jobs by` prompt. 
+
+{{% notice blue "Note" "rocket" %}}
+
+   There are two ways to stop a running app. Either option works so you can pick the one that works best for you!
+   
+   1. Use the IDE.  Click Visual Studio's *Stop* icon.  The light red square that replaces the green *Run* triangle once an app is running.
+   1. Use the terminal. Press *ctrl+C* (a widely-known command to kill a console application). This will work in any terminal context, and not just for our console program in Visual Studio
```

Comment:
```suggestion
   1. In the terminal, pressing *ctrl+C* will also stop the program. This will work in any terminal context, and not just for our console program in Visual Studio
```

Reply:
The original text comes off a little snarky.